std/math.d

```diff
@@ -4406,10 +4418,9 @@ private:
             UNDERFLOW_MASK = 0x10,
             OVERFLOW_MASK  = 0x08,
             DIVBYZERO_MASK = 0x04,
+            SUBNORMAL_MASK = 0x02,
             INVALID_MASK   = 0x01
         }
-        // Don't bother about subnormals, they are not supported on most CPUs.
-        //  SUBNORMAL_MASK = 0x02;
```

Comment:
Comment seems bogus. My run-of-the-mill AMD CPU and the auto-tester ones apparently support it.